python/tank/authentication/ui/login_dialog.py

```diff
@@ -1,20 +1,31 @@
 # -*- coding: utf-8 -*-
+################################################################################
+## Form generated from reading UI file 'login_dialog.ui'
+##
+## Created by: Qt User Interface Compiler version 5.15.2
+##
+## WARNING! All changes made in this file will be lost when recompiling UI file!
+################################################################################
+from .qt_abstraction import QtCore
+for name, cls in QtCore.__dict__.items():
+    if isinstance(cls, type): globals()[name] = cls
```

Comment:
what's that about? This does not look right

Reply:
As I mentioned above, this was added on the recommendation of [pull request #31](https://github.com/shotgunsoftware/tk-flame-review/pull/31/files?diff=split&w=0#diff-3a5749423e737b81cb65521f2192f063873c34a669a4169c20d1cb6250d8746d), which fixes what was done in https://github.com/shotgunsoftware/tk-flame-review/pull/17 of tk-flame. After the tests, it makes sense since when executing build_resources.sh with pyside2 it creates imports in the following way `from PySide2.QtCore import *` therefore QCore is not explicitly used in the code anymore.

Reply:
Ohh I see now what it does. And no, this is incorrect. The flame repo is using this as a way to import Qt/PySide. This is not how we want to do it. Instead, we want to use the tk-core QtImporter like everywhere else.

Reply:
So I think that it can be changed to something like this:
```
from tank.util.qt_importer import QtImporter
importer = QtImporter()
QtCore = importer.QtCore

for name, cls in QtCore.__dict__.items():
    if isinstance(cls, type): globals()[name] = cls
```
why do I insist on `for name, cls ...`? Because, PySide2 
 is generating only `QtSize()` and not `QtCore.QtSize()` (you can check that in the new line 27 of this file)

Reply:
I believe this is because the way classes are available on PySide2 is different from PySide1 which are inside QtCore to mention some.

The code generator doesn't know about our patcher so it thinks `from PySide2 import *` is enough. But this is not ideal for us.

Since we have these classes patched in runtime, I think what Flame and Eduardo did here makes sense. 

Another "clean" way can be to import each of the referenced classes in the file one by one using Qt4 interface. Both ways are going to give us trouble when we want to migrate to Qt5 interface.